ckanext/vocabulary/templates/header.html

```diff
@@ -0,0 +1,110 @@
+{% block header_wrapper %} {% block header_account %}
```

Comment:
@steveoni  I didn't understand what changed on the header template.   Could you explain what change you made here?  I think we should just extend the template instead of overriding the whole template. 
 `{% ckan_extends %}`